skype/skype-ps/skype/New-CsAutoAttendantCallFlow.md

```diff
@@ -117,6 +117,23 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -ForceListenMenuEnabled
+
+If True, Dtmf/Speech inputs will not be processed while the greeting/menu prompt is playing. It will enforce callers to listen to all menu options before making selection.
```

Comment:
```suggestion
If True, DTMF and speech inputs will not be processed while the greeting or menu prompt is playing. It will enforce callers to listen to all menu options before making a selection.
```